scripts/asos/iem_scraper_example.py

```diff
@@ -134,3 +205,37 @@ def main():
 if __name__ == "__main__":
     download_alldata()
     # main()
+
+    
+'''
+Example use case of using the construct_urls and pull_metar functions. 
+
+def main():
```

Comment:
main() is already defined, so you'd want to rename this or remove the previous main()